css/highcharts.scss

```diff
@@ -823,3 +823,13 @@ text.highcharts-drilldown-data-label,
 .highcharts-grid-axis .highcharts-axis-line {
     stroke-width: 1px;
 }
+
+/* NetworkGraph and Packed Bubble */
+.highcharts-simulation-dataLabels-fadeout {
```

Comment:
First, in CSS, don't use camel case. We already have classes like `.highcharts-data-labels`.
Second, I don't think the words _simulation_ and _fadeout_ ready belong in this class name. It should probably be more generic, like `.highcharts-packedbubble-series .highcharts-data-label` and `.highcharts-packedbubble-series .highcharts-data-label-hidden`. Because what we're really define in the CSS is _how_ we show/hide the labels.

Reply:
Corrected, Thank you for catching this :)